doc/en/user/source/community/features-templating/directives.rst

```diff
@@ -1096,9 +1096,20 @@ Notes:
 4) The ``subArray.json`` template (line 6) must be an array itself, the container array will be stripped and
    its values directly integrated inside ``anArray``.
 
-In case of an includeFlat directive is specified and it's attribute value is a property interpolation directive, if the property name evaluates to a json it gets included flat in the final output.
+In case an includeFlat directive is specified and it's attribute value is a property interpolation directive, if the property name evaluates to a json it gets included flat in the final output e.g
```

Comment:
```suggestion
In case an includeFlat directive is specified, and it contains a property interpolation directive evaluating a json, it will get included flat in the final output e.g
```